### PR TITLE
Add a metric(segment/totalCount) about total number of segments on each historical node

### PR DESCRIPTION
**Pull request by tomscut**
#### Added a metric(segment/totalCount) about total number of available segments on each historical node. This metric was emited to ambari metric collector, and could be shown on grafana.
image

<hr>

This PR has:
- [x] been self-reviewed.
- [x] added unit tests or modified existing tests to cover new code paths.
- [x] been tested in a test Druid cluster.
- [x] been running in our production environments.

<hr>

##### Key changed/added classes in this PR
 * `HistoricalMetricsMonitor`
 * `HistoricalMetricsMonitorTest`

##### Key changed/added files in this PR
 * `metrics.md`

### CODE REVIEW

**Comment by tomscut** [comment]
This is useful for monitoring the number of segments on each hitorical node.
image

**Comment by tomscut** [comment]
@jihoonson Hi, could you please review this PR?

**Comment by tomscut** [comment]
This metric is useful for monitoring the number of segments on each hitorical node.

**Comment by tomscut** [comment]
@jihoonson Hi, could you please review this PR? Thx!

File: docs/operations/metrics.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`segment/dropQueue/count`|Number of segments to drop.|server.|Varies.|
 |`segment/size`|Size in bytes of available segments.|dataSource.|Varies.|
 |`segment/count`|Number of available segments.|dataSource.|< max|
+|`segment/totalCount`|Total number of available segments on each historical node.||< max|
```
Comment (jihoonson) [inline docs/operations/metrics.md:209]: Is this a metric for historicals? Then it seems in a wrong section. These are the metrics for the coordinator.

File: server/src/main/java/org/apache/druid/server/metrics/HistoricalMetricsMonitor.java [hunk 2]
```diff
@@ -104,6 +106,8 @@ public boolean doMonitor(ServiceEmitter emitter)
       emitter.emit(builder.build("segment/count", count));
     }
 
+    emitter.emit(new ServiceMetricEvent.Builder().build("segment/totalCount", totalCount));
```
Comment (jihoonson) [inline server/src/main/java/org/apache/druid/server/metrics/HistoricalMetricsMonitor.java:109]: I'm wondering whether the metrics visualization tool can compute this on its side. Is it hard to compute for some reason?

**Pull request closed without merging**